.vscode/settings.json

```diff
@@ -25,19 +25,19 @@
   "[javascriptreact]": {
     "editor.formatOnSave": true,
     "editor.codeActionsOnSave": {
-      "source.organizeImports": true
+      "source.organizeImports": "explicit"
```

Comment:
what does it do?

Reply:
hmm seems booleans are deprecated https://stackoverflow.com/questions/77637621/vscode-workspace-settings-change-on-its-own

Reply:
> The setting value updates are as follows, with the previous boolean values to be deprecated in favor of the string equivalent.
> 
> The options are:
> 
> explicit - Triggers Code Actions when explicitly saved. Same as true.


From https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto